Product.API/Controller/ItemsController.cs

```diff
@@ -0,0 +1,59 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Product.Schema.Models;
+using Product.Service.IServices;
+
+namespace Product.API.Controller
+{
+    [Route("[controller]")]
+    [ApiController]
+    public class ItemsController : ControllerBase
+    {
+        private readonly IItemService _itemService;
+
+        public ItemsController(IItemService itemService)
+        {
+            _itemService = itemService;
+        }
+
+        // GET: api/<ItemsController>
+        [HttpGet("getAll")]
+        public IActionResult GetAllItems()
```

Comment:
Why tenantId is nt passed here?
And it should be also there on database call

Reply:
done